tests/test_tricore.py

```diff
@@ -1,19 +1,28 @@
 #!/usr/bin/env python3
+""" test_tricore.py
+A module for testing tricore lifter.
+"""
+# pylint: disable=wildcard-import
+# pylint: disable=unused-wildcard-import
```

Comment:
Both of these should probably be on the specific line rather than module-level.

Reply:
Done. Also in other modules.